Src/Orbiter/Config.h

```diff
@@ -219,24 +219,25 @@ struct CFG_DEVPRM {
 
 struct CFG_JOYSTICKPRM {
 	DWORD  Joy_idx;				// joystick device index (0=disabled)
-	DWORD  Deadzone;			// central deadzone range for all axes (0-10000)
+	int    Deadzone;			// central deadzone range for all axes (0-10000)
 	DWORD  ThrottleAxis;		// joystick throttle axis (0=none, 1=z-axis, 2=slider 0, 3=slider 1)
```

Comment:
Can these ones move to `int` as well, if we're removing the DWORDS for the others? Or is there a reason to keep them this way?